tests/unit/test_util.cpp

```diff
@@ -53,3 +53,30 @@ TEST_F(UtilTest, toHexWithConsequentZeroes)
     input.push_back(0xab);
     EXPECT_THAT(mococrw::utility::toHex(input), Eq("604569f70011ab"));
 }
+
+TEST_F(UtilTest, testFromHex)
+{
+    std::vector<uint8_t> result;
+    ASSERT_NO_THROW(result = mococrw::utility::fromHex("0x"));
+    ASSERT_TRUE(result.empty());
+
+    ASSERT_NO_THROW(result = mococrw::utility::fromHex("0x1"));
+    ASSERT_TRUE(result == std::vector<uint8_t>{1});
```

Comment:
```suggestion
    ASSERT_THAT(result, Eq(std::vector<uint8_t>{1}));
```
Please use the feature test macros of gtest that give you reasonable output in case of mismatches. Same for all equality tests below.